servicesExtensions/requestSyncrhoneExtensionEscrow_service.py

```diff
@@ -6,13 +6,33 @@
 from servicesCore.requestCore_service import RequestCoreService
 from servicesExternal.web3_single import Web3Single
 
-
+requestCoreArtifact = Artifacts.requestCoreArtifact
```

Comment:
you don't need to do this, they're already imported on line 4 `from artifacts import *`